exporter/containerimage/export.go

```diff
@@ -0,0 +1,228 @@
+package containerimage
+
+import (
+	"bytes"
+	gocontext "context"
+	"encoding/json"
+	"runtime"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/mount"
+	"github.com/containerd/containerd/rootfs"
+	"github.com/moby/buildkit/cache"
+	"github.com/moby/buildkit/cache/metadata"
+	"github.com/moby/buildkit/exporter"
+	"github.com/moby/buildkit/snapshot"
+	"github.com/moby/buildkit/util/flightcontrol"
+	"github.com/moby/buildkit/util/system"
+	digest "github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
```

Comment:
That is unfortunately needed because containerd has not fully standardized to one version.